pkgs/development/libraries/arrayfire/default.nix

```diff
@@ -0,0 +1,26 @@
+{  stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "3.6.1";
+  name = "arrayfire-${version}";
+  src = fetchurl {
+    url = "http://arrayfire.s3.amazonaws.com/3.6.1/ArrayFire-v3.6.1_Linux_x86_64.sh";
+    sha256 = "14qsdbq2yz5bmj5zg4y8sb9pybz2h41zsc3fqv9nhmfmllb004gc";
+  };
+
+  phases = "installPhase";
+
+  installPhase = ''
+    bash ${src} --include-subdirs --prefix=$out
```

Comment:
The source distribution is available from GitHub: https://github.com/arrayfire/arrayfire

This is much preferred compared to this shell unpacker. Please consider repackaging this using the source distribution.

Reply:
I can give it another attempt though. It's possible I messed something up with runtime dependencies for the library.

Reply:
I agree that it doesn't look like a trivial build. I'll remark this for review, and maybe someone else can take a look. As far as I can see, the .sh as built doesn't correctly link dependencies anyway, so I don't think this is a functional approach either.